internal/flagv1/flag_data.go

```diff
@@ -2,13 +2,14 @@ package flagv1
 
 import (
 	"fmt"
-	"github.com/nikunjy/rules/parser"
-	"github.com/thomaspoignant/go-feature-flag/ffuser"
-	"github.com/thomaspoignant/go-feature-flag/internal/utils"
 	"math"
 	"strconv"
 	"strings"
 	"time"
+
+	"github.com/nikunjy/rules/parser"
+	"github.com/thomaspoignant/go-feature-flag/ffuser"
+	"github.com/thomaspoignant/go-feature-flag/internal/utils"
```

Comment:
Can we keep the import as before, there is no change in them just a different order.